crates/re_viewer_context/src/gpu_bridge/tensor_to_gpu.rs

```diff
@@ -483,9 +513,13 @@ fn pad_and_narrow_and_cast<T: Copy + Pod>(
 fn height_width_depth(tensor: &TensorData) -> anyhow::Result<[u32; 3]> {
     use anyhow::Context as _;
 
-    let Some([height, width, channel]) = tensor.image_height_width_channels() else {
+    let Some([mut height, width, channel]) = tensor.image_height_width_channels() else {
         anyhow::bail!("Tensor is not an image");
     };
+    height = match tensor.buffer {
+        TensorBuffer::Nv12(_) => height * 3 / 2,
+        _ => height,
+    };
```

Comment:
isn't `TensorData::image_height_width_channels` already accounting for Nv12 size via the changes you did in `tensor_data_ext`, meaning that we've done it twice now here?

Reply:
oh, right this is the inverse because you want the _texture_ size now, not the image dimensions, right?
In that case, can you please leave a clarifying comment and rename to `texture_height_width_channels`?

Reply:
Yep, exactly. Changed the naming and added a code comment.